source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -772,7 +779,7 @@ The following features of a Connection Pool SHOULD be implemented if they make s
 Background Thread
 ^^^^^^^^^^^^^^^^^
 
-A Pool SHOULD have a background Thread that is responsible for
+A Pool SHOULD have a background Thread (also known as maintenance thread) that is responsible for
```

Comment:
Maybe introducing additional terminology can be avoided here? So we could stick with just `background` naming.

Reply:
I would say, it's better to use maintenance naming everywhere.., but I agree, no need to change it now, reverted